### ISSUE TITLE
Timeout

### ISSUE THREAD

**Issue by AntonHonchar** [alvarotrigo/pagePiling.js#217]
Hello, alvarotrigo.
I've just found one more interesting thing.
So we have - var autoPlayDuration = 1000;
But first slide (when page load ) change much faster than 1000. After that everything is ok.
Do you have any ideas how to fix that? Because it's really to fast (just first slide)

**Comment by AntonHonchar** [comment]
Example of code:
`var interval;
var autoPlayDuration = 1000;

$('#pagePiling').pagepiling({
    sectionsColor: ['yellow', 'orange', '#C0C0C0', '#ADD8E6'],
    loopBottom: true,
    afterLoad: function (anchorLink, index) {
        clearInterval(interval);

        setTimeout(function () {
            interval = getInterval();
        },autoPlayDuration);
    },
    afterRender: function () {
        interval = getInterval();
    }
});

function getInterval() {
    return setInterval(function () {
        $.fn.pagepiling.moveSectionDown();
    }, autoPlayDuration);
}`

**Comment by AntonHonchar** [comment]
http://jsfiddle.net/fzwp74kq/1/

**Comment by victordc55** [comment]
Is this what you wanted ? 

http://jsfiddle.net/s49p8bam/3/

**Comment by AntonHonchar** [comment]
> Is this what you wanted ?
> 
> http://jsfiddle.net/s49p8bam/3/

Not exactly, but I made what I wanted. Here is what I have: https://icradvisory.com

**Comment by alvarotrigo** [comment]
@AntonHonchar consider the Fading Effect extension of fullPage.js for that.
https://alvarotrigo.com/fullPage/extensions/fading-effect.html

**Comment by AntonHonchar** [comment]
> @AntonHonchar consider the Fading Effect extension of fullPage.js for that.
> https://alvarotrigo.com/fullPage/extensions/fading-effect.html

Thank you, Man! Great slider!
